artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/server/impl/RemotingServiceImpl.java

```diff
@@ -537,7 +540,11 @@ public void connectionDestroyed(final Object connectionID) {
       if (conn != null && !conn.connection.isSupportReconnect()) {
          RemotingConnection removedConnection = removeConnection(connectionID);
          if (removedConnection != null) {
-            server.callBrokerPlugins(server.hasBrokerPlugins() ? plugin -> plugin.afterDestroyConnection(removedConnection) : null);
+            try {
+               server.callBrokerPlugins(server.hasBrokerPlugins() ? plugin -> plugin.afterDestroyConnection(removedConnection) : null);
+            } catch (Exception e) {
+               throw new IllegalStateException("Error executing afterDestroyConnection plugin method: " + e.getMessage(), e);
```

Comment:
same here.. I would just log.warn... you cannot interrupt the broker if someone deploys a bugged interceptor.. can you?